### PR TITLE
[TW-318] Refresh "Integrating with Travis CI" page

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Edited and rewrote all text
- Updated tutorial example and verified all steps
- Updated screenshots

### CODE REVIEW

File: src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md [hunk 1]
```diff
@@ -33,69 +34,65 @@ tags:
...
 
-In this example, we’ll walk through how to integrate Postman with [Travis CI](https://travis-ci.org/), a continuous integration service that builds and tests projects on GitHub.
+## Running Postman tests with Travis CI
 
-Travis CI runs your tests every time you commit to your GitHub repo. Then it submits a pull request, or some other specified configuration.
+In general, integrating your [Postman tests](/docs/writing-scripts/test-scripts/) with Travis CI is the same process as [running on Jenkins](/docs/running-collections/using-newman-cli/integration-with-jenkins/), AppVeyor, or any other build system.You will set up your CI configuration to run a shell command when starting your build. The command is a [Newman script](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) that runs the collection containing the tests and returns a pass or fail exit code that’s logged in your CI system.
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md:45]:
```suggestion
In general, integrating your [Postman tests](/docs/writing-scripts/test-scripts/) with Travis CI is the same process as [running on Jenkins](/docs/running-collections/using-newman-cli/integration-with-jenkins/), AppVeyor, or any other build system. You will set up your CI configuration to run a shell command when starting your build. The command is a [Newman script](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) that runs the collection containing the tests and returns a pass or fail exit code that’s logged in your CI system.
```

File: src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md [hunk 2]
```diff
@@ -33,69 +34,65 @@ tags:
 
 ---
 
-Continuous Integration (CI) is a practice that requires developers to integrate code in a shared repository several times a day.
+Continuous Integration (CI) is a development practice that encourages developers to regularly commit their code changes to a shared repository, usually several times a day. Committing early and often helps the team avoid technical debt and detect problems quickly, when conflicts are still relatively easy to fix.
 
-By committing early and often, the team avoids a ton of technical debt by allowing teams to detect problems early while conflicts are relatively easy to fix.
+With CI, every code commit triggers an automated process that builds the code and runs tests. The process can even deploy the updated code if all checks pass, a practice referred to Continuous Delivery (CD).
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md:39]:
```suggestion
With CI, every code commit triggers an automated process that builds the code and runs tests. The process can even deploy the updated code if all checks pass, a practice referred to as Continuous Delivery (CD).
```

❓

File: src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md [hunk 3]
```diff
@@ -33,69 +34,65 @@ tags:
...
 
-1. **Select a Postman collection with tests**: For now, let’s assume you already have a Postman collection with tests. Download the sample collection and environment by clicking the Run in Postman button if you want to follow this example.
-1. **Set up a GitHub repository**: Travis CI is free for open-source projects on GitHub. This example keeps Postman tests in a public GitHub repo.
-1. **Set up Travis CI**: Follow the [Travis CI getting started guide](https://docs.travis-ci.com/user/getting-started) for the complete walk through.
+    [![Run in Postman](https://run.pstmn.io/button.svg)](https://god.gw.postman.com/run-collection/92cc7527bbab2bedffbd?action=collection%2Fimport)
 
-   [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account.
+1. **Set up a GitHub repository.** Travis CI is free for open source projects on GitHub. For this example, you'll store the collection with your Postman tests in a public GitHub repository. Go ahead and create a new public repo on GitHub and clone the project to your local system.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md:57]:
```suggestion
1. **Set up a GitHub repository.** Travis CI is free for open source projects on GitHub. For this example, you'll store the collection with your Postman tests in a public GitHub repository. Create a new public repo on GitHub and clone the project to your local system.
```

File: src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md [hunk 4]
```diff
@@ -33,69 +34,65 @@ tags:
...
-
-    [![tree view tests directory](https://assets.postman.com/postman-docs/travis_tree.png)](https://assets.postman.com/postman-docs/travis_tree.png)
+    ```bash
+    .
+    └── tests
+        └── hello_world.postman_collection.json
+    ```
 
-1. Create a new file called `.travis.yml` and move it to the root of your project repository.
+    > If your collection uses an environment, you should also [export the environment](/docs/sending-requests/managing-environments/) as a JSON file and move it to the `tests` folder as well.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md:73]:
```suggestion
    > If your collection uses an environment, you should also [export the environment](/docs/sending-requests/managing-environments/) as a JSON file and move it to the `tests` folder.
```

Suggest removing either `also` or `as well` here ✂️

File: src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md [hunk 5]
```diff
@@ -106,36 +103,50 @@ Let's learn more about integration with Travis:
...
+    ```bash
+    script:
+    - node_modules/.bin/newman run tests/hello_world.postman_collection.json -e tests/tests.postman_environment.json
+    ```
 
-[![travis fail](https://assets.postman.com/postman-docs/travis_fail.png)](https://assets.postman.com/postman-docs/travis_fail.png)
+1. Commit all the changes to your local project and push them to your public GitHub repo.
 
-   Travis CI is running the Newman command, but you see a failed exit code (1). Boo.
+    Travis CI will automatically trigger a build and run  your tests every time you push a commit to your repo. Head over to the [Travis CI dashboard](https://app.travis-ci.com/dashboard) to see the status of your build. If all went well, your build passed successfully.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/using-newman-cli/integration-with-travis.md:118]:
```suggestion
    Travis CI will automatically trigger a build and run  your tests every time you push a commit to your repo. Navigate to the [Travis CI dashboard](https://app.travis-ci.com/dashboard) to see the status of your build. If all went well, your build passed successfully.
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! Just a few nits, no blockers 🍊

**Pull request merged**